src/components/GetUserLocation.jsx

```diff
@@ -1,11 +1,15 @@
-import { useState } from 'react';
+import { useContext } from 'react';
+import { GeolocationContext } from './GeolocationContextComponent';
 
 export function GetUserLocation() {
-    const [location, setLocation] = useState({})
+
+    const [localLocation, setLocalLocation] = useState({})
+
+    const contextLocationData = useContext(GeolocationContext)
 
     const handleLocationShare = () => {
         window.navigator.geolocation.getCurrentPosition(
-            success => setLocation({latitude: success.coords.latitude, longitude: success.coords.longitude}),
+            success => (setLocalLocation({latitude: success.coords.latitude, longitude: success.coords.longitude}), contextLocationData.updateLocation(localLocation)),
```

Comment:
Hey @ConnieJacques, does this line work correctly? Syntax seems unusual to me (wrapping `()` instead of `{}`, but does it just do those two things at once?

Reply:
I've looked into this, it works. Nice, I didn't know you could combine multiple expressions like this.